app/Nova/Actions/SyncAccess.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Nova\Actions;
+
+use GuzzleHTTP\Client;
+use Illuminate\Bus\Queueable;
+use Laravel\Nova\Actions\Action;
+use Illuminate\Support\Collection;
+use Laravel\Nova\Fields\ActionFields;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+
+class SyncAccess extends Action
+{
+    use InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * Perform the action on the given models.
+     *
+     * @param  \Laravel\Nova\Fields\ActionFields  $fields
+     * @param  \Illuminate\Support\Collection  $models
+     * @return mixed
+     */
+    public function handle(ActionFields $fields, Collection $models)
+    {
+        // This whole function is going to get factored out later
+        // Also it will be a queued job I guess?
+        foreach ($models as $user) {
+            $send = [];
+            $send['uid'] = $user->uid;
+            $send['preferred_first_name'] = $user->preferred_first_name;
```

Comment:
Why not just name it `$send['first_name']` - Does Jedi also have different fields for first/preferred first?